### PR TITLE
EKS Migration staging

### PR DESCRIPTION
**Pull request by njseeto**
[Trello](https://trello.com/c/V7pdyyIn/2075-eks-021-migrate-hmcts-adapter-to-test-dev-and-test-production)

### Add templates to EKS cluster. 
Change ingress file to 'green' and AWS weighting to 0.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -85,6 +93,13 @@ jobs:
             PLATFORM_ENV: production
             K8S_NAMESPACE: hmcts-complaints-formbuilder-adapter-production
           command: './deploy-scripts/bin/deploy'
+      - run:
```
Comment (brenetic) [inline .circleci/config.yml:96]: we do not want to deploy to production yet

Comment (njseeto) [reply .circleci/config.yml]: oh yes, my bad. Have fixed this now

File: .circleci/config.yml [hunk 2]
```diff
@@ -55,6 +55,14 @@ jobs:
...
           command: './deploy-scripts/bin/deploy'
       - slack/status: *slack_status
+      - run:
+          name: deploy to staging (EKS cluster)
+          environment:
+            APPLICATION_NAME: hmcts-complaints-formbuilder-adapter
+            PLATFORM_ENV: staging
+            K8S_NAMESPACE: hmcts-complaints-formbuilder-adapter-staging
+          command: './deploy-scripts/bin/deploy-eks'
+      - slack/status: *slack_status
```
Comment (brenetic) [inline .circleci/config.yml:65]: `slack/status: *slack_status` is always the last thing to happen in a workflow series of steps. it covers any of the run commands above failing. so the one on line 65 is correct but we can therefore remove the one on line 57

Comment (njseeto) [reply .circleci/config.yml:64]: all done!

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**